### PR TITLE
Add component inputs

### PR DESCRIPTION
**Pull request by sah-shah7**
- Added Input fields
image

### CODE REVIEW

**Review by zabihshaik — APPROVED** [review]
Everything looks good to merge.

**Comment by sah-shah7** [comment]
PR approved. Merging

File: Components/Buttons/buttons.html [hunk 1]
```diff
@@ -41,72 +41,72 @@
...
-        </ul>
+              <li class="sidebar-list-items">
+                  <a href="../../Components/toast/toast.html"
+                      class="link-a link-a-font-color-black sidebar-list-items-text">Toast</a>
+              </li>
+              <li class="sidebar-list-items">
+                  <a href="../../Components/Utilities/utilities.html"
+                      class="link-a link-a-font-color-black sidebar-list-items-text">Utilities</a>
+              </li>
+          </ul>
```
Comment (JatuFaique) [inline Components/Buttons/buttons.html:107]: Remove additional spaces

**Review by JatuFaique — COMMENTED** [review]
Everything looks good  to merge

**Review by shivamSspirit — COMMENTED** [review]
good to be merge

**Pull request merged**